app/src/main/java/com/example/poste/activities/Shared_Folder_v2.java

```diff
@@ -45,34 +45,42 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().hide();
 
         // Set the activity layout
-        setContentView(R.layout.activity_shared_folder);
+        setContentView(R.layout.activity_shared_folder_v2);
 
-        TextView folderNameView = findViewById(R.id.share_folder_name);
-        EditText emailView = findViewById(R.id.share_folder_email);
+        TextView folderNameView = findViewById(R.id.textViewShareFolderName);
+        EditText emailView = findViewById(R.id.editTextEmailToShareWith);
+        TextView folderPermissionsView = findViewById(R.id.textViewShareFolderPermissions);
 
-        Spinner spinner = findViewById(R.id.share_folder_spinner);
-        ArrayAdapter<CharSequence> spinnerAdapter = ArrayAdapter.createFromResource(this, R.array.spinner_values, android.R.layout.simple_spinner_item);
-        spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-        spinner.setAdapter(spinnerAdapter);
+        Spinner folderSpinner = findViewById(R.id.share_folder_spinner);
+        Spinner permissionsSpinner = findViewById(R.id.share_folder_permissions_spinner);
+        ArrayAdapter<CharSequence> folderSpinnerAdapter = ArrayAdapter.createFromResource(this, R.array.spinner_values, android.R.layout.simple_spinner_item);
```

Comment:
I think this is why both spinners populate with permissions -- they seem to be built from the same resource